### DISCUSSION TITLE
Ignore images and other media from downloading

### DISCUSSION THREAD

**Discussion by KL13NT** [discordjs/discord.js#6717] [Q&A] [answered]
Hi! 

I have a music bot that plays internet sources. The bot seems to 'hang' whenever someone sends an image, seemingly downloading the `Message` object for the message listener to process. Is there any way to opt out of this? Can I tell the library to ignore media or specific file types in general before they're downloaded?

**Comment by SpaceEEC** [comment, marked as answer]
discord.js doesn't automatically download anything (including attached files).

[`MessageAttachment`](https://discord.js.org/#/docs/main/stable/class/MessageAttachment) only holds metadata that Discord sends along the payload through the gateway.

**Reply by KL13NT** [reply]
Oh, that complicates things. What could possibly be causing the hangup though? I'm trying to trace it but failing drastically.

**Reply by KL13NT** [reply]
Could it be a network slowness issue?
